WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/BlogDashboardPersonalizeCardCell.swift

```diff
@@ -78,7 +78,7 @@ final class BlogDashboardPersonalizeCardCell: DashboardCollectionViewCell {
         }
         WPAnalytics.track(.dashboardCardItemTapped, properties: ["type": DashboardCard.personalize.rawValue], blog: blog)
         let viewController = UIHostingController(rootView: NavigationView {
-            BlogDashboardPersonalizationView(viewModel: .init(service: .init(siteID: siteID)))
+            BlogDashboardPersonalizationView(viewModel: .init(service: .init(siteID: siteID), quickStartType: blog.quickStartType))
```

Comment:
I considered injecting the entire blog, but it would've made it harder to unit-test and create previews for this screen. I don't expect more parameters to be needed.

Reply:
Got it! Thanks for including the diff for an easier review. 

The injection of `quickStartType` was the only question mark for me. However, although at first sight it did catch my attention, I think this approach is more correct even better beyond the argument of making it easier to write unit-tests and create previews. We make `BlogDashboardPersonalizationViewModel` depend on only what it needs to depend on. Due to business requirements, a quick start does require special handling. 

>  I don't expect more parameters to be needed.

Yes, it doesn't look to be needed for the existing cards. In the future, who knows. If more and more custom behaviour would be needed, we could consider some different approach which wouldn't require injecting all the different dependencies into personalization VM and maybe would be handled by some configuration of each card.